Support board view and move button next to original copy link button

SkylineThomasCR opened the pull request:
![image](https://github.com/jaklke/SLC-Copy-Task-To-Clipboard/assets/101559931/2eece24c-2fb9-4525-994f-e41f4d24a81a)

SkylineThomasCR committed:
Support board view and move button next to original copy link button

jaklke commented:
This has an issue
1. open https://collaboration.dataminer.services/squads/68/list
2. open a task
3. in the task details, navigate to the project of this task (by using the breadcrumbs under the title)
4. observe there's no extra button in the task details.

SkylineThomasCR committed:
Trigger on every click

There is no single open task event ID in the list view.
So trigger on every click, but only attempt to add the button a limited number of times.

jaklke committed:
Update main.js
